language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<ol><li>Could you add links to the products or product documentation?</li><li>Also Vertex LLMs should perhaps be "Vertex AI LLMs"</li><li>Could you also elaborate a little more about the value of using LLMs on data in BigQuery? We might not be able to expect everyone coming to this repo will be familiar with Google Cloud products and it'd be good to just give a bit more of a preamble to set the context a bit more (or point to relevant docs).</li><li>I'd also remove the emphasis on "NLP" as I think the terminology has shifted towards "generative AI" as in "applying LLMs to text and other structured data in BigQuery"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>